From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: GSI Maintainer <gsi@localhost>
Date: Tue, 9 Sep 2026 00:00:00 +0000
Subject: [PATCH] Enable automatic brightness by default

Fresh setup always starts at 100% manual brightness, which is
blinding especially in the dark. Default to automatic brightness
mode so the sensor controls it out of the box. User can still
turn it off in Settings > Display.

  def_screen_brightness_automatic_mode false -> true
---
 packages/SettingsProvider/res/values/defaults.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SettingsProvider/res/values/defaults.xml b/packages/SettingsProvider/res/values/defaults.xml
index 8e6e251ff3f2..53324ba4d966 100644
--- a/packages/SettingsProvider/res/values/defaults.xml
+++ b/packages/SettingsProvider/res/values/defaults.xml
@@ -30,7 +30,7 @@
     <bool name="def_accelerometer_rotation">false</bool>
     <!-- Default screen brightness, from 0 to 255.  102 is 40%. -->
     <integer name="def_screen_brightness">102</integer>
-    <bool name="def_screen_brightness_automatic_mode">false</bool>
+    <bool name="def_screen_brightness_automatic_mode">true</bool>
     <fraction name="def_window_animation_scale">100%</fraction>
     <fraction name="def_window_transition_scale">100%</fraction>
     <bool name="def_haptic_feedback">true</bool>
--
2.53.0
